### PR TITLE
Added Clear button and converted the toolbar to Material per #342

### LINKED ISSUE #342: Make the "clear" button on the isochrone more visible
The Isochrone's clear button is pretty easy to miss. Meanwhile, the dashboard has a nicely colored button in the bottom corner for clearing. We should use that kind of Material button on the isochrone for clearing. (And while you're at it, update the isochrone's left-side menu to be all Material-styled.)

### PR DESCRIPTION
**Pull request by SeannyPhoenix**
## Screenshot
Original:
#342 non-Material

After changes:
#342 Material

### CODE REVIEW

**Comment by SeannyPhoenix** [comment]
Updated.

image

File: frontend/src/screens/Isochrone.jsx [hunk 1]
```diff
@@ -553,17 +564,17 @@ class Isochrone extends React.Component {
...
             onChange={this.handleToggleRoute}
             value={route.id}
-          />{' '}
-          {route.id}
-        </label>
-      </div>
+            color="primary"
+          />
+        </ListItemIcon>
+        <ListItemText id={route.id} primary={route.id}/>
```
Comment (exxonvaldez) [inline frontend/src/screens/Isochrone.jsx]: Clicking on the checkbox labels doesn't toggle the checkbox.  There's a way to get a clickable Checkbox label, see https://material-ui.com/components/checkboxes/  for examples.

File: frontend/src/screens/Isochrone.jsx [hunk 2]
```diff
@@ -625,49 +636,64 @@ class Isochrone extends React.Component {
...
+              </Grid>
+            </Grid>
+            <Grid container
+              className="isochrone-controls"
+              direction="column"
+              alignItems="left">
+              <Grid item>
+                <Typography variant="subtitle1" color="primary">Routes</Typography>
+              </Grid>
+              <Grid containter item
```
Comment (exxonvaldez) [inline frontend/src/screens/Isochrone.jsx]: I'm seeing some warnings in the browser console about the attributes being passed into Grid -- container is misspelled here.

**Comment by SeannyPhoenix** [comment]
I would like to break the scrolling problem into its own issue; this problem is actually inherited from before.

File: frontend/src/screens/Isochrone.jsx [hunk 3]
```diff
@@ -625,49 +637,64 @@ class Isochrone extends React.Component {
...
-                  <span onClick={this.selectNoRoutesClicked}>none</span>
-                </div>
-                <div className="isochrone-routes">
+                </Select>
+              </Grid>
+            </Grid>
+            <Grid container
+              className="isochrone-controls"
+              direction="column"
+              alignItems="left">
```
Comment (exxonvaldez) [inline frontend/src/screens/Isochrone.jsx]: I'm seeing a warning (always good to keep the developers tools open while developing) in the console:  index.js:1437 Warning: Failed prop type: Invalid prop `alignItems` of value `left` supplied to `ForwardRef(Grid)`, expected one of ["flex-start","center","flex-end","stretch","baseline"].

File: frontend/src/screens/Isochrone.jsx [hunk 4]
```diff
@@ -625,49 +637,64 @@ class Isochrone extends React.Component {
...
-            <div className="isochrone-controls">
-              <div>
-                stat:
-                <select
+          <Control position="topleft" className="">
+            <Grid container
+              className="isochrone-controls"
+              direction="column">
+              <Grid item>
+                <Typography variant="subtitle1" color="primary">Statistic</Typography>
```
Comment (exxonvaldez) [inline frontend/src/screens/Isochrone.jsx]: The primary color is meant for the primary action in a dialog or card.  So the primary color here throws me off, I keep expecting them to be clickable.  Can you keep these heading on the default color (no color attribute)?

**Comment by exxonvaldez** [comment]
> I would like to break the scrolling problem into its own issue; this problem is actually inherited from before.

Sounds good.   Above the relevant pull request comment, you can click the `... -> reference in new issue` to create a new issue.  I'll do it this time.

**Comment by exxonvaldez** [comment]
> Looks good to merge when you're ready! Thanks for making the fixes.

Oh, I see @akgupta89  asked for a multiple select.  Maybe we can create a new issue for that?  Feels like this PR should go in so it doesn't sit around too much longer.

**Pull request merged**